modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/configuration/RocksDbDataRegionValidatorImpl.java

```diff
@@ -51,6 +52,6 @@ public void validate(RocksDbDataRegionName annotation, ValidationContext<String>
     }
 
     private static boolean contains(RocksDbStorageEngineView engineConfig, String dataRegion) {
-        return engineConfig.defaultRegion().name().equals(dataRegion) || engineConfig.regions().get(dataRegion) != null;
+        return true;
```

Comment:
Why?

Reply:
I removed the rocksdb region ahead of time. Removed this hack and will remove the whole class in the next PR, I think :)